### ISSUE TITLE
Save as new sets published to True

### ISSUE THREAD

**Issue by glogiotatidis** [mozmeao/snippets-service#560]
Clicking `save as new` when editing a snippet always sets `published` property to `True`.

This is a regression that appeared after #546

**Comment by mediaChick** [comment]
@glogiotatidis When do you think this issue will move from Review to Complete?

**Comment by mediaChick** [comment]
Clicking "save as new" when editing a snippet still sets published property to True.

**Comment by jgmize** [comment]
#561 was merged into master but not yet into production: https://github.com/mozmeao/snippets-service/compare/production...master

**Comment by jgmize** [comment]
@mediaChick the fix in #561 has now been deployed into production; would you mind verifying that it works correctly now?

**Comment by mediaChick** [comment]
@jgmize It does! Thank you!
